tests/derive.test.tsx

```diff
@@ -303,24 +304,33 @@ it('basic underive', async () => {
 describe('glitch free', () => {
   it('basic (#296)', async () => {
     const state = proxy({ value: 0 })
-    const derived1 = derive({
-      value: (get) => get(state).value,
-    })
-    const derived2 = derive({
-      value: (get) => get(derived1).value,
-    })
+    const derived1 = derive(
+      { value: (get) => get(state).value },
+      { sync: false }
+    )
+    const derived2 = derive(
+      { value: (get) => get(derived1).value },
+      { sync: false }
+    )
     const computeValue = jest.fn((get) => {
       const v0 = get(state).value
       const v1 = get(derived1).value
       const v2 = get(derived2).value
-      return v0 + (v1 - v2)
-    })
-    const derived3 = derive({
-      value: (get) => computeValue(get),
+      return `v0: ${v0}, v1: ${v1}, v2: ${v2}`
     })
+    const derived3 = derive(
+      {
+        value: computeValue,
+      },
+      { sync: false }
+    )
 
+    let renderCount = 0
     const App = () => {
       const snap = useSnapshot(derived3)
+
+      renderCount++
```

Comment:
We should count commits not renders. Let me tweak the test.

Reply:
Oh very interesting I didn't know useEffect was the way to count commits! I learned a lot here thanks!